aspnetcore/release-notes/aspnetcore-9/samples/PAR/Program.cs

```diff
@@ -0,0 +1,27 @@
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Authentication.OpenIdConnect;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// <snippet_1>
+builder.Services
+    .AddAuthentication(options =>
+    {
+        options.DefaultScheme = CookieAuthenticationDefaults.AuthenticationScheme;
+        options.DefaultChallengeScheme = OpenIdConnectDefaults.AuthenticationScheme;
+    })
+    .AddCookie()
+    .AddOpenIdConnect("oidc", oidcOptions =>
+    {
+        // Other provider-specific configuration goes here.
+
+        // The default value is PushedAuthorizationBehavior.UseIfAvailable.
+        oidcOptions.PushedAuthorizationBehavior = PushedAuthorizationBehavior.Disable;
+    });
```

Comment:
@josephdecock I get 'OpenIdConnectOptions' does not contain a definition for 'PushedAuthorizationBehavior' and no accessible extension method 'PushedAuthorizationBehavior' accepting a first argument of type 'OpenIdConnectOptions' could be found
Even with RC1 installed. I can see [PushedAuthorizationBehavior.Require](https://source.dot.net/#Microsoft.AspNetCore.Authentication.OpenIdConnect/PushedAuthorizationBehavior.cs,3af73de8f33b70c5)